tools/make/usage.txt

```diff
@@ -4,7 +4,6 @@ Usage: make <cmd>
   make help                   Print this message.
   make notes                  Search for code annotations.
   make list-files             List files.
-  make install                Run install sequence.
```

Comment:
Given that `make install` could be used for things beyond just running `npm install`, you can keep this around.